tests/h/settings_test.py

```diff
@@ -3,98 +3,104 @@
 from __future__ import unicode_literals
 
 import pytest
-import mock
+import logging
 
 from h import settings
 
 
-class FakeSetting(object):
-    def __init__(self, result):
-        self.result = result
+def asutf8(string):
+    return string.encode()
 
-    def __call__(self, environ):
-        return self.result
 
-    def __str__(self):
-        return 'fake setting'
-
-
-class TestDeprecatedSetting(object):
-    def test_emits_warnings_when_child_setting_is_used(self):
-        func = settings.DeprecatedSetting(FakeSetting(result={'foo': 'bar'}),
-                                          message='what to do instead')
-        func.warn = mock.Mock(spec_set=[])
-
-        result = func({})
-
-        assert result == {'foo': 'bar'}
-        func.warn.assert_called_once_with('use of fake setting is '
-                                          'deprecated: what to do instead')
+def test_database_url():
+    url = 'postgres://postgres:1234/database'
+    expected = 'postgresql+psycopg2://postgres:1234/database'
```

Comment:
I guess this function is already there on master, but it seems to be a function that isn't used at all. We should send a separate quickie PR to delete it